tests/stress/km/stress_tests_km.cpp

```diff
@@ -1048,6 +1050,283 @@ _print_test_control_info(const test_control_info& test_control_info)
     }
 }
 
+static void
+_cleanup_tailcall_program(const fd_t& prog_map_fd)
+{
+    // Cleanup tail calls
```

Comment:
```suggestion
    // Clean up tail calls.
```
nit: "clean up" is two words as a verb (one word as a noun).